code/game/gamemodes/meteor/meteors.dm

```diff
@@ -190,7 +192,7 @@
 	pass_flags = PASSTABLE | PASSGRILLE
 	hits = 1
 	hitpwr = 3
-	meteorsound = 'sound/weapons/throwtap.ogg'
+	meteorsound = 'sound/weapons/Gunshot_smg.ogg'
```

Comment:
Kind of a placeholder to replace throw-tap without reverting to the station rocking boom. hard to find a noise that fist everything space dust hits from metal to glass to Robusten.